tests/sles4sap/robot_fw.pm

```diff
@@ -49,6 +72,9 @@ sub run {
     foreach my $robot_test (split /\n/, script_output "ls $test_repo") {
         record_info("$robot_test", "Starting $robot_test");
         script_run "robot --log $robot_test.html --xunit $robot_test.xml $robot_test";
+        # Soft fail section
```

Comment:
I think it would be a good idea to expand this comment to roughly explain how to add more soft fails into the code.

Reply:
indeed, I was about to raise the question about that :smile: